test/conformance/conformance_global_control.cpp

```diff
@@ -431,3 +432,9 @@ TEST_CASE("terminate_on_exception: enabled") {
     CHECK(terminate_handler_called);
 }
 #endif
+
+TEST_CASE("custom assertion handler type") {
```

Comment:
[brief_scan](https://github.com/intel-innersource/libraries.threading.infrastructure.onetbb-ci/actions/runs/17268695145/job/49007273481#logs) reports here
`ERROR: conformance_global_control.cpp:435: found TEST_CASE without brief`

Reply:
I missed that one. Thanks! Added.